### PR TITLE
Use hexadecimal for printing numeric thread IDs on NetBSD; misc. CI updates

### PR DESCRIPTION
**Pull request by johnsonjh**
* Use hexadecimal for printing numeric thread IDs on NetBSD
* Fixes for running 32-bit binaries in CI container environment

### CODE REVIEW

File: include/sir/config.h [hunk 1]
```diff
@@ -93,12 +93,23 @@
 # define SIR_PIDSUFFIX ")"
 
 /**
- * The format for the current process/thread ID.
+ * The format for the current process ID.
  *
  * @remark Only applies if ::SIRO_NOPID or ::SIRO_NOTID are not set.
```
Comment (aremmell) [inline include/sir/config.h:98]: If you split them then this comment is inaccurate now; should just be NOPID

File: include/sir/config.h [hunk 2]
```diff
@@ -93,12 +93,23 @@
...
  */
 # define SIR_PIDFORMAT "%d"
 
+/**
+ * The format for the current thread ID.
+ *
+ * @remark Only applies if ::SIRO_NOPID or ::SIRO_NOTID are not set.
```
Comment (aremmell) [inline include/sir/config.h:105]: Ditto for this one

**Review by aremmell — APPROVED** [review]
We are gonna have merge conflicts but 🤷🏻‍♂️

File: include/sir/config.h (hunk 1 above)
Comment (johnsonjh) [reply include/sir/config.h:98]: I'll fix

File: include/sir/config.h [hunk 3]
```diff
@@ -93,12 +93,23 @@
...
+ * @remark Only applies if ::SIRO_NOPID is not set.
  */
 # define SIR_PIDFORMAT "%d"
 
+/**
+ * The format for the current thread ID.
+ *
+ * @remark Only applies if ::SIRO_NOTID is not set.
+ */
+# if defined(__NetBSD__)
```
Comment (aremmell) [inline include/sir/config.h:107]: why don't we change this to `__USE_HEX_TIDS__`, and define it in platform so we don't have to put all the platforms in this header? thoughts?

thinking ahead to when we have 4-5 here

Comment (johnsonjh) [reply include/sir/config.h:107]: Sure ...

**Review by aremmell — APPROVED** [review]

**Pull request merged**